python/foglamp/tasks/purge/purge.py

```diff
@@ -50,8 +49,8 @@ class Purge(FoglampProcess):
             "default": "72"
         },
         "size": {
-            "description": "Maximium size of data to be retained, the oldest data will be removed to keep below this size," +
-                           "unless retained. (in Hours)",
+            "description": "Maximum size of data to be retained, the oldest data will be removed to keep below this "
+                           "size, unless retained. (in Bytes)",
```

Comment:
As per **FOGL-737**, The size will be defined in Kbytes. However I asked to @MarkRiddoch about doc updation here https://docs.google.com/document/d/1GdMTerNq_-XQuAY0FJNQm09nbYQSjZudKJhY8x5Dq8c/edit#heading=h.vml5x5f6ih6

Reply:
not sure about this one. we can take it later as found.